api/hasura/event-triggers/utils/handleCheckNomineeMsg.ts

```diff
@@ -0,0 +1,30 @@
+import { gql } from '../../../../api-lib/Gql';
+import { sendSocialMessage } from '../../../../api-lib/sendSocialMessage';
+import { EventTriggerPayload } from '../../types';
+
+export default async function handleCheckNomineeMsg(
+  payload: EventTriggerPayload,
+  channels: { discord?: boolean; telegram?: boolean }
+) {
+  const {
+    event: { data },
+  } = payload;
+
+  if (
+    data.old.ended === false &&
+    data.new.ended === true &&
+    new Date(data.new.expiry_date) < new Date()
+  ) {
+    const { nominees_by_pk } = await gql.getNominee(data.new.id);
+
+    if (nominees_by_pk) {
+      await sendSocialMessage({
+        message: `Nominee ${nominees_by_pk.name} has only received ${nominees_by_pk.nominations_aggregate?.aggregate?.count} vouch(es) and has failed`,
+        circleId: nominees_by_pk.circle_id,
+        channels,
+      });
+      return true;
+    }
+  }
+  return false;
```

Comment:
Instead of wrapping the main logic in an if block and returning false at the end, it would be easier to follow if you returned early by inverting the if condition and then put the main logic after that.

```suggestion
  const shouldSendMessage = 
    data.old.ended === false &&
    data.new.ended === true &&
    new Date(data.new.expiry_date) < new Date();
    
  if (!shouldSendMessage)  return false;

  const { nominees_by_pk } = await gql.getNominee(data.new.id);

  if (!nominees_by_pk) return false;
  
   await sendSocialMessage({
      message: `Nominee ${nominees_by_pk.name} has only received ${nominees_by_pk.nominations_aggregate?.aggregate?.count} vouch(es) and has failed`,
      circleId: nominees_by_pk.circle_id,
      channels,
    });
  return true;
```